.github/workflows/ci.yml

```diff
@@ -428,7 +428,7 @@ jobs:
         run: echo "::add-matcher::.github/problem-matcher.json"
       - name: Cleanup node
         # This is required as a virtual environment update 20210219.1 left too little space for MemSQL to work
-        if: matrix.modules == ':trino-memsql'
+        if: matrix.modules == 'plugin/trino-memsql'
```

Comment:
could this use pattern matching or substring searching?

Reply:
Yes

Reply:
Both `contains` and `endsWith` are available in GHA expressions.

Reply:
awesome, let's do contains `trino-mysql`, thanks